src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/LocationIntegrationTest.kt

```diff
@@ -79,7 +79,7 @@ class LocationIntegrationTest: IntegrationTest() {
           operationalCapacity = 0,
           userDescription = "",
           internalLocationCode = "",
-          locationUsage = "",
+          locationUsage = null,
```

Comment:
you're calling the constructor with named parameters so don't need to include nulls in the constructor